.github/workflows/ci.yml

```diff
@@ -801,14 +810,21 @@ jobs:
             --env "BUILD_PRESET=benchmark-with-tracing" \
             --env "IREE_HOST_BIN_DIR=${BUILD_DIR}/install/bin" \
             "${DOCKER_IMAGE}" "${BUILD_SCRIPT}" "${BUILD_TOOLS_DIR}/build-traced"
+      - name: "Unpacking tracy capture"
+        env:
+          TRACY_CAPTURE_ARCHIVE: ${{ steps.download-tracy-capture.outputs.tracy-capture-archive }}
+        run: |
+          tar -C "${BUILD_TOOLS_DIR}/build-traced" \
+            -xvf "${TRACY_CAPTURE_ARCHIVE}"
       - name: "Creating the benchmark tools archive"
         id: archive
         env:
-          BENCHMARK_TOOLS_ARCHIVE: ${{ matrix.target.platform }}-${{ matrix.target.arch }}-benchmark-tools.tar
+          BENCHMARK_TOOLS_ARCHIVE: ${{ env.PLATFORM }}-${{ env.ARCH }}-benchmark-tools.tar
         run: |
           tar -cf "${BENCHMARK_TOOLS_ARCHIVE}" \
             "${BUILD_TOOLS_DIR}"/*/tools/iree-benchmark-module \
-            "${BUILD_TOOLS_DIR}"/*/tools/build_config.txt
+            "${BUILD_TOOLS_DIR}"/*/tools/build_config.txt \
```

Comment:
It's a little bit tricky to download the tool in `.github/workflows/benchmark_execution.yml` because the platform runs `iree-benchmark-module` and `tracy-capture` can be on different (e.g. Android benchmarks run tracy-capture on RPi but benchmarking on mobile phones). The pipeline `benchmark_execution.yml` is driven by the config file exported from benchmark suites. If we want to put `tracy-capture` info in the config file, it basically puts the details of how we set up CI benchmark machines into the benchmark suites, which I don't think is right.

The second place to download it is in `build_tools/benchmarks/run_benchmarks.sh`. I think it's the best place to fetch the `tracy-capture`. The problem is we don't want to have network communication everywhere?

So I decided to pack everything into each benchmark tool archive to simplify the problem. Let me know if you have other ideas.

Reply:
Urgh I guess that makes sense. It still seems pretty gross. We're also going to be making a ton of copies of the same file, though hopefully GCS is smart enough to not actually store it a bunch of times. Could you at least put a note saying why we're doing it this way?

Reply:
Done.